packages/apps/docs/src/components/Markdown/UnorderedList/styles.css.ts

```diff
@@ -0,0 +1,20 @@
+import { sprinkles, vars } from '@kadena/react-ui/theme';
+
+import { globalStyle, style } from '@vanilla-extract/css';
+
+export const ulListClass = style([
+  sprinkles({
+    marginY: '$5',
+    marginX: 0,
+    position: 'relative',
+  }),
+]);
+
+// should work, but doesn't
+// globalStyle(`${ulListClass} li > ul`, {
+globalStyle(`article ul > li >ul`, {
```

Comment:
this concerns me because it is only targeting native elements :/ Can you try `${ulListClass} > li > ul`?

Reply:
@eileenmguo - updated! please review it again.